Packs/RSANetWitness_v11_5/Integrations/RSANetWitness115/RSANetWitness115.yml

```diff
@@ -0,0 +1,1401 @@
+    - contextPath: RSANetWitness115.FileAlertsList.distinctAlertCount.low
+      type: Number
+  - arguments:
+    - default: false
+      description: Agent ID of the host.
+      isArray: false
+      name: agent_id
+      required: true
+      secret: false
+    - default: false
+      description: Service ID of the Endpoint Server to be connected.
+      isArray: false
+      name: service_id
+      required: false
+      secret: false
+    - default: false
+      description: Path where the files may be present, either specify a single file
+        path or use a wildcard. for example - "C:\\Users\\sample\\*"
+      isArray: false
+      name: path
+      required: true
+      secret: false
+    - default: false
+      description: Maximum number of files returned by the host matching the wildcard
+        path (default 10).
+      isArray: false
+      name: count_files
+      required: false
+      secret: false
+    - default: false
+      description: Maximum size of each file (in MB) when using a wildcard path (default
+        100 MB).
+      isArray: false
+      name: max_file_size
+      required: false
+      secret: false
+    deprecated: false
+    description: Initiate file download for a single file, or multiple files, to the
+      Endpoint Server
+    execution: false
+    name: rsa-nw-file-download
+  - arguments:
+    - default: false
+      description: Agent ID of the host.
+      isArray: false
+      name: agent_id
+      required: true
+      secret: false
+    - default: false
+      description: Service ID of the Endpoint Server to be connected.
+      isArray: false
+      name: service_id
+      required: false
+      secret: false
+    deprecated: false
+    description: initiatesthe download of MFT to the Endpoint Server.
+    execution: false
+    name: rsa-nw-mft-download-request
+  - arguments:
+    - default: false
+      description: Agent ID of the host.
+      isArray: false
+      name: agent_id
+      required: true
+      secret: false
+    - default: false
+      description: Service ID of the Endpoint Server to be connected.
+      isArray: false
+      name: service_id
+      required: false
+      secret: false
+    deprecated: false
+    description: initiate the download of the system dump to the Endpoint Server.
+    execution: false
+    name: rsa-nw-system-dump-download-request
+  - arguments:
+    - default: false
+      description: Agent ID of the host.
+      isArray: false
+      name: agent_id
+      required: true
+      secret: false
+    - default: false
+      description: Service ID of the Endpoint Server to be connected.
+      isArray: false
+      name: service_id
+      required: false
+      secret: false
+    - default: false
+      description: ID of the process.
+      isArray: false
+      name: process_id
+      required: false
+      secret: false
+    - default: false
+      description: Identifier of the process in windows.
+      isArray: false
+      name: eprocess
```

Comment:
```suggestion
      name: windows_process_id
```

Reply:
I can change it but it shows as eprocess in context, what do you think?